api/tests/unit/domain/models/AuthenticationMethod_test.js

```diff
@@ -232,5 +232,42 @@ describe('Unit | Domain | Models | AuthenticationMethod', function () {
         ).to.throw(ObjectValidationError);
       });
     });
+
+    context('GARAuthenticationComplement', function () {
+      let validArguments;
+      beforeEach(function () {
+        validArguments = {
+          firstName: 'Margaret',
+          lastName: 'Remington',
+        };
+      });
+
+      it('should successfully instantiate object when passing all valid arguments', function () {
+        // when
+        expect(() => new AuthenticationMethod.GARAuthenticationComplement(validArguments)).not.to.throw(
+          ObjectValidationError
+        );
+      });
+
+      it('should throw an ObjectValidationError when firstName is not valid', function () {
+        // when
+        expect(
+          () => new AuthenticationMethod.GARAuthenticationComplement({ ...validArguments, firstName: 1234 })
+        ).to.throw(ObjectValidationError);
+        expect(
+          () => new AuthenticationMethod.GARAuthenticationComplement({ ...validArguments, firstName: undefined })
+        ).to.throw(ObjectValidationError);
+      });
```

Comment:
**chipotage**
On a eu un peu de mal à comprendre les 2 expect avec Andreia. 
Ca aurait été plus clair en 2 it différents : `should throw an ObjectValidationError when firstName is not on correct format` et `should throw an ObjectValidationError when firstName is not defined`.

Reply:
J'avoue que j'ai copié-collé les tests des autres classes AuthenticaionComplement.
Ça correspond en fait aux deux règles de validation JOI `string` et `required`, donc on teste que ça échoue si la propriété est autre chose qu'une string, ou absente.
Mais OK pour ajouter des  `it`, ça permettra de le préciser.